README.md

```diff
@@ -1,13 +1,4 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
+This repo contains Charles Floeder's portfolio and STEP projects.
```

Comment:
You can probably replace "Charles Floeder's" with "my" (since it's on your github page)

Reply:
I know it's mine when I'm looking at it. However, it may be helpful for other viewers.